docs/docs/howto/workload-deployment/TLS-configuration.md

```diff
@@ -35,7 +35,7 @@ The following example is for an application with these properties:
 Add the following annotations to your workload:
 
 ```yaml
-metadata: # apps/v1.Deployment, apps/v1.DaemonSet, ...
+metadata: # v1.PodTemplate
```

Comment:
This might benefit from additional explanation above, like

> Add the following annotations to your Pod, or to the pod template metadata of your controller.